.github/workflows/build.yml

```diff
@@ -1,11 +1,31 @@
 name: build and push image
+
 on:
   pull_request:
   push:
     branches:
       - azul
+
 env:
-  REGISTRY: ghcr.io
+  # The fully qualified image name of the image to build. The first component is
+  # the host name of the registry that primarily hosts the image.
+  #
+  azul_docker_bigquery_emulator_image: docker.io/ucscgi/azul-bigquery-emulator
+
+  # An optional prefix for the image name if the image is to be pushed to an
+  # alternate registry instead of the one noted in the image name above.
+  #
+  azul_docker_registry: ""
+
+  # The version where this repo diverges from the upstream repo.
+  #
+  azul_docker_bigquery_emulator_upstream_version: 0.4.4
+
+  # The version of this image. The image will be tagged with the version of the
+  # upstream image followed by the value of this variable. Increment this value
+  # to update the OS packages installed in the image.
```

Comment:
The last sentence isn't currently true. We need the apt-get update upgrade stuff in the Dockerfile.

We should also pin the Debian base image tag in the Dockerfile and add a variable for that here.